docs/usage.rst

```diff
@@ -1,34 +1,38 @@
 Using ``pyramid_nacl_session``
 ==============================
 
-Generating a Random Secret
---------------------------
+Installation
+------------
 
-First, you need to arrange to have a 32-byte random secret available:
+Install using setuptools, e.g. (within a virtualenv)::
 
-.. code-block:: python
+  $ pip install pyramid_nacl_session
 
-   from pyramid_nacl_session import generate_secret
+Setup
+-----
 
-   SECRET = generate_secret(as_hex=False)
+Once :mod:`pyramid_nacl_session` is installed, you typically use the 
+``config.include`` mechanism to include it into your Pyramid project's 
+configuration. In your Pyramid project's ``__init__.py``:
 
-Configure a Session Factory
----------------------------
+.. code-block:: python
 
-Use the generated secret to configure a session factory:
+   config = Configurator(.....)
+   config.include('pyramid_nacl_session')
```

Comment:
Five dots is not syntactically correct and looks weird. An actual example would be fine, and better to use a context manager with `with`.

```
    with Configurator(settings=settings) as config:
        config.include('pyramid_nacl_session')
        # ... more config.includes
        config.scan()
```